digiwf-engine/digiwf-engine-service/src/main/resources/prozesse/example/digiwf-kennenlernen/element-templates/EmailIntegration.json

```diff
@@ -0,0 +1,216 @@
+        "type": "camunda:in",
+        "expression": true,
+        "target": "receiversCc"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Receiver (BCC)",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "expression": true,
+        "name": "receiversBcc",
+        "target": "receiversBcc"
+      },
+      "constraints": {
+        "notEmpty": false
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "${receiversBcc}",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "receiversBcc"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Subject",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "name": "subject",
+        "target": "subject"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Body",
+      "type": "Text",
+      "value": "",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "expression": true,
+        "name": "body"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "${body}",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "body"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Reply-To Address",
+      "type": "String",
+      "value": "",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "expression": true,
+        "name": "replyTo",
+        "target": "replyTo"
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "${replyTo}",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "replyTo"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Attachment Paths (S3)",
+      "type": "String",
+      "value": "",
+      "description": "Array of presigned urls created with s3 integration",
+      "binding": {
+        "type": "camunda:inputParameter",
+        "expression": true,
+        "name": "attachments",
+        "target": "attachments"
+      },
+      "constraints": {
+        "notEmpty": false
+      }
+    },
+    {
+      "type": "Hidden",
+      "value": "${attachments}",
+      "binding": {
+        "type": "camunda:in",
+        "expression": true,
+        "target": "attachments"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Dispatch Status",
+      "value": "mailSentStatus",
+      "type": "String",
+      "binding": {
+        "type": "camunda:out",
+        "source": "mailSentStatus"
+      },
+      "constraints": {
+        "notEmpty": false
+      }
+    }
+  ]
+}
```

Comment:
Das Gleiche, wie

https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-engine/digiwf-engine-service/src/main/resources/prozesse/example/email-example-V02/element-templates/EmailIntegration.json

und

https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-engine/digiwf-engine-service/src/main/resources/prozesse/example/email-example-V02/element-templates/EmailIntegration.json

Reply:
O.k. ich habe die Element Templates gelöscht und checke die nicht mit ein